R/find_statistic.R

```diff
@@ -0,0 +1,162 @@
+#' @title Find statistic for model
+#' @description Returns the statistic for the a regression model (*t*-statistic,
+#'   *z*-statistic, etc.).
+#' @name find_statistic
+#'
+#' @description Small helper that checks if a model is a regression model
+#'   object and return the statistic used.
+#'
+#' @param x An object.
+#' @param ... Currently not used.
+#'
+#' @return A character describing the type of statistic. If there is no
+#'   statistic available with a distribution, a `NULL` will be returned.
+#'
+#' @examples
+#' data(mtcars)
+#' m <- lm(mpg ~ wt + cyl + vs, data = mtcars)
+#'
+#' find_statistic(m)
+#' @export
+find_statistic <- function(x, ...) {
+
+  # model object check --------------------------------------------------------
+
+  # check if the object is a model object; if so, quit early
+  if (isFALSE(is_model(x))) {
```

Comment:
I would not use `isFALSE()` because it was implemented rather late in R and we want to be more backwards compatible than just R 3.5 or 3.6.